src/features/utils/instrument-base.js

```diff
@@ -139,6 +139,8 @@ export class InstrumentBase extends FeatureBase {
         return hasReplayPrerequisite(this.agentIdentifier) && !!session
       case FEATURE_NAMES.sessionTrace:
         return !!session
+      case FEATURE_NAMES.logging:
+        return !!session
```

Comment:
While this makes sense and I can see why you added this bit, product decided that the logging feature should still import and run if session is disabled.